File: Java programming/problems on numbericals/problem5.java
/*
5. Accept one number from user and print that number of * on screen.  

	Author: Milind Nalawade
	
	Date:18/06/2021
	
	language : java 14.0.2
*/

import java.util.*;
import java.lang.*;

class problem5
{
	public static void main(String args[])
	{
		Scanner sobj=new Scanner(System.in);
		System.out.print("Enter your number : ");
		int no1=sobj.nextInt();
		
		while(no1!=0)
	{
		System.out.print("* ");
		no1--;
	}
	}
}

File: C Programming/problem on numbericals/problem5.c
/*
5. Accept one number from user and print that number of * on screen.  
	
	Author: Milind Nalawade
	
	Date:18/06/2021
	
	language : C programming
*/
#include<stdio.h>

//Entry point function

int main()
{
	int no=0;
	printf("Enter your number : ");
	scanf("%d",&no);
	
	while(no)
	{
		printf("* ");
		no--;
	}
}

